src/main/java/lotto/domain/DefaultLottoGenerator.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.List;
+import lotto.view.OutputView;
+
+public class DefaultLottoGenerator implements LottoGenerator {
+    private static final int MIN_NUMBER = 1;
+    private static final int MAX_NUMBER = 45;
+    private static final int LOTTO_NUMBER_SIZE = 6;
+
+    @Override
+    public Lottos generateLottos(int count) {
+        List<Lotto> lottos = new ArrayList<>();
+        for (int i = 0; i < count; i++) {
+            List<Integer> randomNumbers = generateRandomNumber();
+            lottos.add(new Lotto(randomNumbers));
+            OutputView.printLottoNumbers(randomNumbers);
+        }
+        return new Lottos(lottos);
+    }
```

Comment:
의미를 알릴 수 없는 `i` 대신 다른 변수명을 사용하는 것은 어떨까요?
```suggestion
    @Override
    public Lottos generateLottos(int totalCount) {
        List<Lotto> lottos = new ArrayList<>();
        for (int count = 0; i < totalCount; count++) {
            List<Integer> randomNumbers = generateRandomNumber();
            lottos.add(new Lotto(randomNumbers));
            OutputView.printLottoNumbers(randomNumbers);
        }
        return new Lottos(lottos);
    }
```

또한 반복적으로 반복적으로 출력하는 경우 StringBuilder 혹은 BufferedWriter 클래스의 사용하는 것도 고려해보면 좋을 것 같아요!